src/main/webapp/status.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en" ng-app="sectionsApp">
+
+<head>
+<meta http-equiv="Content-Type" content="text/html; Charset=UTF-8">
+<title>Canvas Course Manager Status Page</title>
+<link href="bootstrap/css/bootstrap.css" type="text/css"
```

Comment:
This URL does not resolve - try:
<link href="vendor/bootstrap/bootstrap.min.css" type="text/css"
	rel="stylesheet" media="all">